tasks.rb

```diff
@@ -47,7 +52,17 @@ def display_usage(command_list)
   # Apply rubocop
   cops = ['Layout',
```

Comment:
Aligns with OS-HPXML?

Reply:
Yep. Helps to catch potential bugs (like the `corridor_position` bugfix in this case).